livekit-agents/livekit/agents/llm/llm.py

```diff
@@ -21,6 +21,10 @@ class LLMMetrics(TypedDict):
     duration: float
     label: str
     cancelled: bool
+    completion_tokens: int
```

Comment:
not sure if already discussed but time to first token also interesting

Reply:
it is already here, it is called ttft